app/helpers/works_helper.rb

```diff
@@ -150,10 +150,10 @@ def download_url_for_work(work, format)
     url_for("#{path}?updated_at=#{work.updated_at.to_i}").gsub(' ', '%20')
   end
 
-  # Generates a list of a work's tags and details for use in feeds
+ # Generates a list of a work's tags and details for use in feeds
```

Comment:
I think the changes in this file just got picked up when merging with master and are unrelated.